root/defaults/geoip2.conf

```diff
@@ -0,0 +1,83 @@
+## Version 2020/09/20 - Changelog: https://github.com/linuxserver/docker-swag/commits/master/root/defaults/geoip2.conf
+# To enable, uncommment the Geoip2 config line in nginx.conf
+# Add the -e MAXMINDDB_LICENSE_KEY=<licensekey> to automatically download the Geolite2 database.
+# A Maxmind license key can be acquired here: https://www.maxmind.com/en/geolite2/signup
+# The below config is for geoblocking any country/city you add. The default config is blocking all countries/cities except the ones you set to "yes". 
+# If you want to do the opposite, set default to "yes", and the countries/cities to "no".
+# As the default config blocks all IP's except the ones in the GeoLite2-City.mmdb database set to yes, you will need to allow your lan ip for local access. 
+
+
+geoip2 /config/geoip2db/GeoLite2-City.mmdb {
+    auto_reload 5m;
+    $geoip2_data_country_code country iso_code;
+    $geoip2_data_city_name city names en;
+}
+
+# GEOIP2 COUNTRY CONFIG
+map $geoip2_data_country_code $allowed_country {
+    default no;
+    <COUNTRY-CODE> yes; # e.g GB yes; for United Kingdom
+    IP/CIDR yes; # e.g. 192.168.1.0/24 yes; for local access.
+}
+
+#(Optional)
+# GEOIP2 CITY CONFIG
+#    map $geoip2_data_city_name $allowed_city {
+#    default no;
+#    <CITY-NAME> yes; # e.g Inverness yes;
+#    IP/CIDR yes; # e.g. 192.168.1.0/24 yes; for local access.
+#    }
+
+
+# Server config example:
```

Comment:
Can you include these comments in the default site config and mention viewing this file for usage instructions
```nginx
# COUNTRY GEO BLOCK	
#	if ($allowed_country = no) {
#	return 444;
#	}
```

Then change the rest of what's left here (currently called Full Example) to explain how to add it to *other* server blocks such as proxy subdomain configs?

I think that's easier.

Reply:
Yeah, good idea